aws/eks/k8s_rbac.tf

```diff
@@ -0,0 +1,18 @@
+resource "kubernetes_cluster_role_binding" "view-rolebinding" {
+  count = local.cluster_features["k8s_rbac_view"] ? 1 : 0
```

Comment:
Does this essentially disable this entire block when count=0?

Reply:
yeah, it's a pretty common tf pattern to enable feature flags.  You can see it being used a bunch of times in the helm chart https://github.com/mozilla-it/terraform-modules/blob/master/aws/eks/helm.tf#L22